### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by PedroH-Neves**
Here I created two new pages: 

- Home
- Quote

Using routes to make these new pages to work in a javascript single page model.

### CODE REVIEW

File: src/components/NavBar.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+.navBar-title {
+  font-size: 46px;
+}
+
+.navBar-ul {
+  display: flex;
+  list-style: none;
+  gap: 10px;
+  font-size: 20px;
+}
```
Comment (OyePriscilla) [inline src/components/NavBar.css:17]: - [ ] Please kindly add the vertical line in-between the `nav-items` as we have it in the given template. I'm sure one of the essence of this milestone is to learn how to reproduce a given design, hiring managers are also on a look out for this quality as well.

@PedroH-Neves 
## Wireframe
crr

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
 
 const App = () => (
-  <div>
-    <Calculator />
-  </div>
+  <Router>
+    <NavBar />
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/Calculator" element={<Calculator />} />
```
Comment (OyePriscilla) [inline src/App.js:14]: - [ ] I kindly suggest you style this page according to the wireframe as well. 

## Wireframe 
png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @PedroH-Neves 😃, 

You've really done a great work, I must confess!

There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Git Flow observed✔️
Linters Set up correctly ✔️ 
Descriptive PR summary and title ✔️ 

## Required Changes ♻️

- [ ] Please kindly add a working link to the live demo section of your README file, if it's not available at the moment; kindly indicate **`NOT AVAILABLE AT THE MOMENT`**. If you implement the suggestion listed above, I'm so sure that your README file will look more professional and appealing. 👍  Thanks. 🙏 

## Optional suggestions

- [optional] **Please ensure to write a more descriptive branch name for subsequent PRs, this will give an overview of what you've done so far in the milestone. Thanks. 🙏**  

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/NavBar.css (hunk 1 above)
Comment (rloterh) [reply src/components/NavBar.css:17]: resolved! ✔️

File: src/App.js (hunk 2 above)
Comment (rloterh) [reply src/App.js:14]: resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Neves,  

Your project is complete! 🏁 Good job! There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- SPA feature properly implemented!
- Good ReadMe!

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**